src/v2/Apps/Consign/Routes/MarketingLanding/Components/__tests__/ReadMore.jest.tsx

```diff
@@ -33,6 +33,25 @@ describe("ReadMore", () => {
   it("tracks clicks on articles", () => {
     const wrapper = getWrapper()
     const articles = wrapper.find("Article")
+
+    const article = articles.first()
+    expect(article.props().href).toBeDefined()
+    article.simulate("click")
+
+    expect(trackEvent.mock.calls[0]).toMatchInlineSnapshot(`
+      Array [
+        Object {
+          "action": "clickedArticleGroup",
+          "context_module": "relatedArticles",
+          "context_page_owner_type": "consign",
+          "destination_page_owner_id": "5e6fc79cb77f450020738595",
+          "destination_page_owner_slug": "artsy-editorial-decide-consign-art",
+          "destination_page_owner_type": "article",
+          "type": "thumbnail",
+        },
+      ]
+    `)
```

Comment:
I'm wondering what would be the best approach here...

The previous existent tests are still passing with the new implementation:
https://github.com/artsy/force/blob/a34d0f92af576f0e062e7db6b5cfc5f6352dedcb/src/v2/Apps/Consign/Routes/MarketingLanding/Components/__tests__/ReadMore.jest.tsx#L55-L69

I wasn't able to replace this with the `toMatchInlineSnapShot()` version because I can't use this matcher inside a loop.

I added this one here, but it's only checking for a single element instead of all of the 6 that are present.

Which solution would be better in this scenario?
1) Checking a single element with the `.toMatchInlineSnapshot()` matcher
2) Checking the 6 elements with the old implementation
3) Some other possibility that I'm not thinking of

Reply:
good question - I think either would be OK! Regardless of approach, we're confirming that tracking is being called and we're passing parameters with the right shape, which is what's important. So feel free to go with either one.

Reply:
Cool!! I'm keeping the previously existent test then!!